cmd/control_loop/manifest_reader/manifest_reader.go

```diff
@@ -0,0 +1,43 @@
+package manifest_reader
```

Comment:
I don't see a strong reason to have this reader outside of the controller package. The manifest is an input to the controller and (conceptually) is tightly bound to the controller. Looking ahead to possibly integrating this into the registry tool, I think it would make sense to have all of these files (including apis.go, specs.go, etc) in a single "controller" package that is sourced from the related command - with the exception of main.go, which would go into the cmd/registry/cmd package (following existing patterns).

Reply:
Done.

Moved `manifest_reader` for now. Will move the ret of them later while integrating with the registry tool.